autograder/cli/submission/fetch-attempts.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+
+import autograder.api.submission.fetchattempts
```

Comment:
Order is off.

Reply:
Looking at other files, I am unsure about what the correct order should be.

Reply:
Sorry, I read the code wrong.
This is correct.